voluptuous/schema_builder.py

```diff
@@ -1244,16 +1244,22 @@ class Remove(Marker):
     [1, 2, 3, 5, '7']
     """
 
+    def __init__(
+        self,
+        schema_: Schemable,
+        msg: typing.Optional[str] = None,
+        description: typing.Optional[str] = None,
+    ) -> None:
+        super().__init__(schema_, msg, description)
+        self.__hash__ = lru_cache(maxsize=None)(lambda: object.__hash__(self))
```

Comment:
Why is this necessary? Doesn't `Marker.__hash__` cover this?

Reply:
Its overriding the hash here to use the base `object.__hash__` instead of `Marker.__hash__`